pages/feed/index.tsx

```diff
@@ -111,7 +124,19 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
         image={props?.ogImage}
         twitterCard="summary_large_image"
       />
-      <div className="container flex flex-col gap-12 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row">
+      <div className="container flex flex-col gap-16 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row">
+        <div className="flex-col flex-1 hidden gap-8 mt-12 md:flex">
+          <div>
```

Comment:
Why we must keep this empty div, What happens when the user is not logged??? Please could you put this check on the top of this wrapper?